spec/models/instrument_spec.rb

```diff
@@ -47,5 +47,41 @@
 
         expect(instruments).to eq(expected)
       end
+
+      it 'Can search by name' do
+        dealer = create(:dealer, id: 1)
+        guitar1 = create(:instrument, dealer_id: 1, name: "A")
+        guitar2 = create(:instrument, dealer_id: 1, name: "C")
+        guitar3 = create(:instrument, dealer_id: 1, name: "B")
+
+        guitar = Instrument.search("B")
+        expected = [guitar3]
+
+        expect(guitar).to eq(expected)
+      end
+
+      # it 'Raise error is nothing found' do
+      #   dealer = create(:dealer, id: 1)
+      #   guitar1 = create(:instrument, dealer_id: 1, name: "A")
+      #   guitar2 = create(:instrument, dealer_id: 1, name: "C")
+      #   guitar3 = create(:instrument, dealer_id: 1, name: "B")
+      #
+      #   guitar = Instrument.search("D")
+      #   expected = [guitar3]
+      #
+      #   expect(guitar).to eq(expected)
+      # end
```

Comment:
@duguy8 do you still need this?

Reply:
@abreaux26 Good catch, don't need this just forgot to take it out